.github/quarkus-github-bot.yml

```diff
@@ -285,13 +285,17 @@ triage:
         - extensions/resteasy-classic/resteasy-qute/
     - id: reactive-messaging
       labels: [area/reactive-messaging, area/smallrye]
-      title: "reactive.messaging"
+      expression: |
+        matches("reactive.messaging", title)
+        || matches("Mqtt", titleBody)
+        || matches("RabbitMQ", titleBody)
+      titleBody: "reactive.messaging"
       notify: [cescoffier, ozangunalp]
       directories:
         - extensions/smallrye-reactive-messaging
     - id: rest-client
       labels: [area/rest-client]
-      title: "rest.client"
+      titleBody: "rest.client"
```

Comment:
I'm not sure we want to do that. We used to have a lot of `titleBody` and they ended up being a real pain.

Typically, in this case if someone is copy pasting the Quarkus startup with the list of extensions, it will trigger the rule.

Reply:
If you guys found that there where lots of false positives that is sensible

Reply:
Seems I have missed this one last evening